### PR TITLE
Fix/app link

### PR DESCRIPTION
**Pull request by palmfjord**
We should probably utilize the app link in the header

### CODE REVIEW

File: src/utils/tracking/trackEvent.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export const trackEvent = (
```
Comment (hedvigoscar) [inline src/utils/tracking/trackEvent.ts]: I'd suggest using `quepasa` for this instead 🙂

Comment (palmfjord) [reply src/utils/tracking/trackEvent.ts]: sure! is it ready tho? we kinda need to get this out as we're shipping traffic to the wrong place now 🙈

Comment (hedvigoscar) [reply src/utils/tracking/trackEvent.ts]: You can do this for a while, but you should consider replacing it very soon (`quepasa` should be stable enough this week)

Comment (palmfjord) [reply src/utils/tracking/trackEvent.ts]: amaze! ping me when its ready? :D

**Pull request merged**